packages/oui-progress-tracker/_mixins.less

```diff
@@ -81,20 +86,37 @@
       }
     }
 
-    &__step_complete {
-      cursor: pointer;
-      pointer-events: initial;
+    &__label {
+      position: relative;
+      top: rem-calc(40);
```

Comment:
The relative positionning is necessary here ?

--position: relative;
--top: rem-calc(40);
++margin-top: rem-calc(40);

Reply:
Its been removed :)